### DISCUSSION TITLE
Publish assets (linux head)

### DISCUSSION THREAD

**Discussion by HakanL** [unoplatform/uno#11132] [Q&A]
I'm trying to use the new Uno v4.7 solution structure with a shared .NET project (love it!), but I'm running into problems publishing it to my Linux system. It runs fine, but none of the images are displayed (they are all entered in this format in the xaml: ms-appx:///Shared/Assets/Images/logo.png, the shared project is called "Shared"). The images are there in the build folder, but not in the publish folder. I've added a csproj item to manually copy them, but that doesn't seem right. Am I missing something?

**Comment by FrozDark** [comment]
Set build action of assets to Content

**Reply by jeromelaban** [reply]
This is fixed in https://github.com/unoplatform/uno/pull/11057, and will be backported to 4.7 very soon.

**Reply by HakanL** [reply]
> Set build action of assets to Content

They are, and they are copied to the build output, but not the published output.
